content/player.fr.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Lecteurs multimédia"
+title_image: /content-images/wai-media-guide/player-nobackground.svg
+nav_title: "Lecteur multimédia"
+
+lang: fr   # change "en" to lang code, here and 2 @@s below
+last_updated: 2020-10-28   # Change to date of translation YYYY-MM-DD (month in middle)
+# translator: "..."
+Translators:
+-name: Sofia Ahmed
+# contributors: "..."
+
+permalink: /media/av/player/fr   # Add lang to end /link/to/page/@@
+ref: /media/av/player/   # Do not change this
+layout: default
+github:
+   repository: w3c/wai-media-guide
+   path: 'content/player.fr.md'   # Add lang to the middle of the filename, e.g., index.@@.md
+
+resource:
+  ref: /media/av/
+navigation:
+  previous: /media/av/av-content/
+  next: /media/av/description/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Introduit les considérations à prendre en compte pour fournir un lecteur multimédia qui accepte l'accessibilité.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour 10 septembre 2019. JOURNAL DES MODIFICATIONS.</p>
```

Comment:
We should not translate "CHANGELOG" as it is automatically translated in the process.